File: clients/typescript/src/SystemdManagerGetUnitReply.ts
import {SystemdUnit} from './SystemdUnit';
export class SystemdManagerGetUnitReply {
  private _unit: SystemdUnit;

  constructor(input: {
    unit: SystemdUnit,
  }) {
    this._unit = input.unit;
  }

  get unit(): SystemdUnit { return this._unit; }
  set unit(unit: SystemdUnit) { this._unit = unit; }
}

File: clients/rust/src/systemd_manager_get_unit_reply.rs
// SystemdManagerGetUnitReply represents a SystemdManagerGetUnitReply model.
#[derive(Clone, Debug, Deserialize, Eq, Hash, Ord, PartialEq, PartialOrd, Serialize)]
pub struct SystemdManagerGetUnitReply {
    #[serde(rename="unit")]
    pub unit: Box<crate::SystemdUnit>,
}

impl SystemdManagerGetUnitReply {
    pub fn new(unit: crate::SystemdUnit) -> SystemdManagerGetUnitReply {
        SystemdManagerGetUnitReply {
            unit: Box::new(unit),
        }
    }
}
